upgrade/README.md

```diff
@@ -56,7 +56,48 @@ The upgrade is a guided process starting with [Upgrade Management Nodes and CSM
 - Although it is not recommended, the [Booting CSM `barebones` image](../operations/validate_csm_health.md#5-booting-csm-barebones-image)
   test may be skipped if all compute nodes are active running application workloads.
 
-See [Validate CSM Health](../operations/validate_csm_health.md).
+1. (`ncn-m002#`) If a typescript session is already running in the shell, then first stop it with the `exit` command.
+
+1. (`ncn-m002#`) Start a typescript.
+
+    ```bash
+    script -af /root/csm_upgrade.$(date +%Y%m%d_%H%M%S).post_upgrade_health_validation.txt
+    export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+    ```
```

Comment:
@alexanderkingh Did you want to make this more generic wrt what you were proposing for recording timing information? Perhaps your calls to start / stop a stage can also start / stop typescript? If you are not ready for it yet then nvm. However, I don't think we would want multiple instructions to start / stop logging + metrics.

Reply:
Thanks. The name of the typescript at this point is unique enough that we can key off of to record the timing. We'll just match on post_upgrade_health_validation being put into the console. Otherwise this looks fine to me.

Reply:
If we're going to have some kind of "start stage" and "stop stage" thing that we have people run at the beginning/end of stages, then I'd be fine with building in the typescript logic in there. I just wanted to start collecting typescript data for upgrades, since for some reason we never included instructions to do that, even though we always have for installs. It makes debugging issues much easier.